recipes/opene57/all/conanfile.py

```diff
@@ -8,7 +8,7 @@
 from conan.tools.microsoft import is_msvc, is_msvc_static_runtime
 from conan.tools.scm import Version
 
-required_conan_version = ">=1.53.0"
+required_conan_version = ">=2.0.5"
```

Comment:
```suggestion
required_conan_version = ">=1.54.0"
```

Please, keep Conan 1.x compatibility. All recipes in CCI are still compatible.